modules/ROOT/pages/salesforce-to-twilio.adoc

```diff
@@ -0,0 +1,454 @@
+= Create Your First Mule Application with Flow Designer
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+
+You are the connectivity lead for Salesforce Sales Cloud at Northern Trail Outfitters (NTO). NTO makes stylish and comfortable apparel, and gear for outdoor recreation and fitness.
+
+NTO uses Sales Cloud to track the opportunities in the pipeline for bulk swag orders. After they close opportunities, sales reps need to go into the order management system and place the orders.  Also, their customers often call NTO to check on orders.
+
+NTO wants to use MuleSoft to create a process that places orders automatically when opportunities are closed, and that notifies customers via SMS about their order status and ETA.
+
+Your task is to put this process in place within a tight deadline. You realize that it is critical for four systems that NTO uses to be integrated with each other for this process to work
+
+[%header,cols=2*]
+|===
+|System
+|Purpose
+
+|Salesforce Sales Cloud
+|Manages opportunities
+
+|MySQL database
+|Stores product information and SKUs
+
+|NTO Order system
+|Internal system that can place orders and return tracking information + order ETA. Callable through the NTO Order API.
+
+|Twilio
+|Sending notifications to customers
+|===
+
+== Your approach
+To automate this process, you have to overcome a few integration challenges. You come up with this list of what needs to happen in order to automate this order to cash cycle.
+
+. Salesforce triggers a message to start the process when the status of an opportunity is changed to “Closed Won”.
```

Comment:
Salesforce starts the workflow automation process when a sale is made. This is triggered in Salesforce when an opportunity is changed to "Closed Won".

Reply:
Nice revision!